CHANGELOG.md

```diff
@@ -36,6 +36,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixed bugs
 
+* Fix confusing/unclear error message, when next/prev can not find a target commit
+  by explicitly specifying the commit where the search for the target started from.
```

Comment:
nit: I wouldn't add changelog entry because users don't have to know error/warning messages are improved.

Reply:
Removed.